### PR TITLE
Renamed summary functions for table 1

### PR DESCRIPTION
**Pull request by cschaerfe**

### CODE REVIEW

**Comment by cschaerfe** [comment]
not quite ready to merge. I still need to update the documentation

File: R/get_tableone.R [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
 #' @details It is possible to provide your own summary function. Please have a loot at summary for inspiration.
-#' 
+#'
 #' @note All columns in the table will be summarized. If only some columns shall be used, please select only those
 #' variables prior to creating the summary table by using dplyr::select()
-#' 
+#'
 #' @examples
-#' 
+#'
```
Comment (SHAESEN2) [inline R/get_tableone.R:24]: remove library call from examples

File: R/tableone.R [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
 #'
-#' @examples 
-#' 
+#' @examples
+#'
 #' library(dplyr)
 #' library(visR)
-#' 
-#' 
+#'
```
Comment (SHAESEN2) [inline R/tableone.R:24]: remove lib calls

File: tests/testthat/test-utils_table.R [hunk 3]
```diff
@@ -1,177 +1,176 @@
...
-#' Rebecca Albrecht
+#' @section Last updated by: Charlotta Fruechteniccht
 #' @section Last update date:
-#' 20-APR-2021
+#' 29-MAY-2021
 
 # Specifications ----------------------------------------------------------
 
-#' T1. Correct values for summarize.numeric
+#' T1. Correct values for summarize_long.numeric
```
Comment (SHAESEN2) [inline tests/testthat/test-utils_table.R:8]: define correct

**Review by SHAESEN2 — CHANGES_REQUESTED** [review]

File: R/get_tableone.R (hunk 1 above)
Comment (cschaerfe) [reply R/get_tableone.R]: done

File: R/tableone.R (hunk 2 above)
Comment (cschaerfe) [reply R/tableone.R]: done

File: tests/testthat/test-utils_table.R (hunk 3 above)
Comment (cschaerfe) [reply tests/testthat/test-utils_table.R:8]: correct is defined in the subtasks

**Comment by cschaerfe** [comment]
@timtreis , @SHAESEN2 , @bailliem - I think I'm done here now. Please merge whenever you found time to review

**Review by SHAESEN2 — APPROVED** [review]

**Pull request merged**